uw-frame-components/css/buckyless/helpers.less

```diff
@@ -1,58 +0,0 @@
-/* General Helper Classes */
```

Comment:
How do I think about things like this disappearing in the spirit of [Semantic Versioning](http://semver.org/)? Losing styles can be not-backwards-compatible as experienced by frame-based apps, such that they'd have to change stuff upon upgrade to a version of frame with this changeset? Does that mean this changeset should bump the `MAJOR` version of frame?